pom.xml

```diff
@@ -12,16 +12,18 @@
         <description>IUCDS pilot AWS Middleware</description>
 
         <parent>
-                <groupId>com.cooldatasoft</groupId>
+                <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-starter-parent</artifactId>
-                <version>2.2.5.RELEASE</version>
+                <version>2.3.0.RELEASE</version>
         </parent>
 
         <properties>
                 <docker.image.prefix>nhsd</docker.image.prefix>
                 <java.version>11</java.version>
                 <aws-sdk.version>1.11.362</aws-sdk.version>
                 <jacoco.line.coveredRatio>0.65</jacoco.line.coveredRatio>
+                <spotbugs.version>4.0.0</spotbugs.version>
+                <maven.project.info.reports.plugin.version>2.9</maven.project.info.reports.plugin.version>
```

Comment:
you dont need "maven.project.info.reports.plugin.version", this should be coming from the parent pom after updating the parent